templates/base.html

```diff
@@ -163,12 +163,12 @@
           <li><a href="{{ config_set['BRANDING']['footer_url'] }}" ng-safenewtab><img src="{{ config_set['BRANDING']['footer_img'] }}"></a></li>
           {% endif %}
           <li quay-require="['BILLING']"><a href="https://docs.quay.io" ng-safenewtab>Documentation</a></li>
-          <li quay-require="['BILLING']"><a href="/tos" target="_self">Terms</a></li>
-          <li quay-require="['BILLING']"><a href="/privacy" target="_self">Privacy</a></li>
+          <li quay-require="['BILLING']"><a href="https://www.openshift.com/legal/terms">Terms</a></li>
+          <li quay-require="['BILLING']"><a href="https://www.redhat.com/en/about/privacy-policy">Privacy</a></li>
```

Comment:
#242 fixes these correctly. Ensure that this PR doesn't modify the target after #242 is merged into master.